dtree.c

```diff
@@ -661,15 +743,53 @@ void handleTextInput(SDL_Event *event){
     }
 }
 
+void moveCursorLine(int relative_line){
```

Comment:
`relative_line` is an int, but it seems like the only two values you expect it to have are `-1` and `1` for moving up and down. Perhaps a boolean for up versus down would be better here? At the same time maybe reworking the function to jump to any line (e.g. `moveCursorToLine(int line_num)` could be more useful in general (e.g. `moveCursorToLine(0)` to jump to the first line, `cur_line-1` or `cur_line+1` for up/down, etc)

Reply:
That was the original plan, but it turned out to be harder than I anticipated, so I instead hardcoded up and down. I should change it though if I rework the cursor.